Automattic/wp-calypso
# Skip initial form for saved reports

Fixes https://github.com/Automattic/dotcom-forge/issues/7615

## Proposed Changes

* skip the initial form for saved reports
* show the message  "Getting your report" on the loading screen

## Why are these changes being made?


* We're updating the Site Profiler UI

## Testing Instructions



* Go to a saved report URL, eg. `/site-profiler/`
* Check that the initial form is skipped
* Check that the loading screen only displays the message "Getting your report..."
* Check with a new URL for regressions


https://github.com/Automattic/wp-calypso/assets/11555574/16b0acda-799b-4734-9e34-5cacd31b844e



## Pre-merge Checklist
